plugins/AuthMe/config.yml

```diff
@@ -94,13 +94,13 @@ settings:
         # The next time the player joins the server, if his IP
         # is the same as last time and the timeout hasn't
         # expired, he will not need to authenticate.
-        enabled: false
+        enabled: true
```

Comment:
Khá thiếu an toàn, nhất là khi server đã public với VPS ngoài rồi

Reply:
okay :v